### PR TITLE
Add ecommerce to Docker

### PR DESCRIPTION
**Pull request by michaelroytman**

### CODE REVIEW

**Comment by jibsheet** [comment]
👍

File: playbooks/roles/ecommerce/tasks/main.yml [hunk 1]
```diff
@@ -109,6 +136,9 @@
     - "collectstatic --noinput"
     - "compress"
   when: not devstack
+  tags:
+    - assets
+    - assets:gather
 
 - name: Write out the supervisor wrapper
   template:
```
Comment (fredsmith) [inline playbooks/roles/ecommerce/tasks/main.yml:144]: can you tag the rest of these tasks as well?

these should probably be install:configuration

File: playbooks/roles/ecommerce/tasks/main.yml [hunk 2]
```diff
@@ -141,10 +171,16 @@
...
+    - install:configuration
 
 - name: Update supervisor configuration
   shell: "{{ supervisor_ctl }} -c {{ supervisor_cfg }} update"
   when: not disable_edx_services
+  tags:
+    - manage
+    - manage:start
 
 - name: Create symlinks from the-er venv bin dir
```
Comment (fredsmith) [inline playbooks/roles/ecommerce/tasks/main.yml:194]: install:configuration for this one
install:base for symlinks into the virtualenv.

**Comment by fredsmith** [comment]
👍

**Pull request merged**